src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,77 @@
-import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  todoList: Todo[];
+  selectedTodo?: Todo | null;
+  showTodoDetails: (value: number) => void;
+};
 
-    <tbody>
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">1</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">delectus aut autem</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
+export const TodoList: React.FC<Props> = ({
+  todoList,
+  selectedTodo,
+  showTodoDetails = () => {},
```

Comment:
The default value for `showTodoDetails` is unnecessary since this function is always provided by the parent component. Consider removing this default to simplify the code.